### PR TITLE
Psychopath->Uncaring

### PR DESCRIPTION
**Pull request by DukePaulAtreid3s**
#### Summary
Interface "Renamed the ‘Psychopath’ mutation to ‘Uncaring’ "

#### Purpose of change
Many people dislike the psychopath trait for, among other reasons, incorrectly representing a real-world mental condition. This PR makes it a more generic “Uncaring”,  instead of being a b-movie-esque “Psychopath”.

#### Describe the solution
Changed the description and name, did not alter ID

#### Describe alternatives you've considered
Not doing this, and/or removing some of the incredibly edgy “Psychopath” dialogue.

### CODE REVIEW

**Comment by Maleclypse** [comment]
My vote is renaming it to Edgelord with the description “All edge!”

**Comment by fairyarmadillo** [comment]
Psychopathy is a real thing and worth modeling. Instead of renaming it and reinforcing the fact that it's a freebie trait that lets you ignore annoying mechanics, it should be more like the real thing. There is not a single accepted definition but IMO things the trait should do:
- Make you not care about what happens to other people or their remains
- Reduce fear responses in some cases, but not all
- Reduce the character's ability to persuade, increase their ability to lie and intimidate. Drawbacks are important to prevent this trait from just being something everyone grabs to ignore mechanics. #70797 also introduces some potentially attractive traits which are mutually exclusive with Psychopath, so part of that work is already done.

And we should go over any interactions with cannibal/spiritual/etc to ensure that what psychopathy is doing is reducing emotional response to witnessing or performing harm to others, and not being The Joker of New England.

If there's edgelord content we want to keep for gameplay reasons, it can be ported over to Killer Drive.

**Comment by DukePaulAtreid3s** [comment]
> Psychopathy is a real thing and worth modeling. Instead of renaming it and reinforcing the fact that it's a freebie trait that lets you ignore annoying mechanics, it should be more like the real thing. There is not a single accepted definition but IMO things the trait should do:

So the issue people have with it is moreso that this trait is a freebie, and not the naming and flavoring of it?

**Comment by fairyarmadillo** [comment]
> So the issue people have with it is moreso that this trait is a freebie, and not the naming and flavoring of it?

It's just my opinion. 

For comparison, we didn't rename Kaluptic Psychosis because it wasn't politically correct, it was renamed because it's nothing like real schizophrenia. If we could model real schizophrenia, it would be called that. Psychopath is within spitting distance of a recognized real-life condition and just needs a little cleaning up. This was not the case with Kaluptic Psychosis.

**Comment by DukePaulAtreid3s** [comment]
> > So the issue people have with it is moreso that this trait is a freebie, and not the naming and flavoring of it?
> 
> 
> 
> It's just my opinion. 
> 
> 
> 
> For comparison, we didn't rename Kaluptic Psychosis because it wasn't politically correct, it was renamed because it's nothing like real schizophrenia. If we could model real schizophrenia, it would be called that. Psychopath is within spitting distance of a recognized real-life condition and just needs a little cleaning up. This was not the case with Kaluptic Psychosis.

Interesting, I suppose, if the people upstairs don't like this change, desensitized can become a different trait. My main inspiration for this re-fluff was how some military personell (especially snipers), have coping mechanisms to deal with the inherently... personal? Nature of their killing. Humans aren't designed to shoot each other in the back. Perhaps lacking guilt about killing, but having a passive (significant) morale penalty would work for something like that. I'm not sure though.

**Comment by fairyarmadillo** [comment]
> My main inspiration for this re-fluff was how some military personell (especially snipers), have coping mechanisms to deal with the inherently... personal? Nature of their killing.

That's already in the game. There are proficiencies you can start with that negate the mood debuff from certain actions that reflect professional training, and you have a hidden counter that ticks up when you do bad things that makes you eventually stop feeling bad about them.

The trait that exists is not at all the same thing. It represents a diminished ability to feel empathy or certain emotions, along with increased aggression and risk-taking. It is a psychological issue and not some killer's mantra.

**Comment by fairyarmadillo** [comment]
Excuse me, I thought this was an issue and not a pull request.

Since it's a pull request I gotta say this is the worst possible fix, if a fix is even needed (and I don't think one is). The name doesn't work because there's already an unrelated desensitizing system, making it out to be some kind of special training in the description is even edgier and less realistic than it already is, and saying that something is an offensive depiction and then simply renaming it is really only hiding it and not helping anybody.

**Comment by DukePaulAtreid3s** [comment]
Damn ok then

**Comment by I-am-Erk** [comment]
Who is "we" in this case, fairyarmadillo? You weren't part of those discussions. Not wanting to stigmatize actual illnesses was, in fact, one of the reasons behind that change, albeit far from the only one. I'm very much in favour of it as a general rule.

In general, may I request that you take care in how you word things so that you don't appear to be talking for the dev team? You're fine to voice your opinions and understanding, but you're phrasing this as though you're part of a larger group. It's something I've seen you do before. 


In this case, while "desensitized" probably isn't the right term, the reasoning behind removing "psychopath" is very reasonable. Feel free to debate a better term, but don't make your arguments as though you speak for "the people upstairs" (whoever they are, I'm on the ground floor right now)

**Comment by fairyarmadillo** [comment]
> Who is "we" in this case, fairyarmadillo? You weren't part of those discussions.

First off, I literally just said

> It's just my opinion.

> In general, may I request that you take care in how you word things so that you don't appear to be talking for the dev team? You're fine to voice your opinions and understanding, but you're phrasing this as though you're part of a larger group. It's something I've seen you do before.

I'm referring to the community of people who work on the game, which I am a part of.

> Not wanting to stigmatize actual illnesses was, in fact, one of the reasons behind that change, albeit far from the only one. I'm very much in favour of it as a general rule.

I'm paraphrasing what you've said in the past. I can dig up the quote if you want.

**Comment by esotericist** [comment]
personally, i'm in support of renaming _and_ removing the unnecessary edgelordery. imo the messaging belongs, at best, in something like crazy cataclysm.

**Comment by I-am-Erk** [comment]
Look, what you wrote was:

> For comparison, we didn't rename Kaluptic Psychosis because...

I don't care that you stated in another paragraph that something was your opinion, this makes it sound like you have anything to do with this decision, as clearly evidenced by muad dib here thinking you spoke for the dev team, something you made no effort to correct. This lead to you convincing someone to close a PR, despite you having frankly *no idea* whether or not it's something the merge team wants to see.

Don't do that, and don't debate it. Just don't.

**Comment by fairyarmadillo** [comment]
> personally, i'm in support of renaming _and_ removing the unnecessary edgelordery. imo the messaging belongs, at best, in something like crazy cataclysm.

Switching it to something like Low Empathy might work. I'd just be wary of making it out to be some kind of fantasy thing and then just straight up depicting a real-life condition.

Kaluptic Psychosis is a bit different because it's so far off from schizophrenia IRL. But like if someone called the game out for depicting obesity in a negative light and we simply renamed it to Apocalyptic Adipose Syndrome that would not be a fix.

**Comment by DukePaulAtreid3s** [comment]
> Switching it to something like Low Empathy might work. I'd just be wary of making it out to be some kind of fantasy thing and then just straight up depicting a real-life condition.

Funnily enough, you just read my mind.

**Comment by DukePaulAtreid3s** [comment]
> personally, i'm in support of renaming _and_ removing the unnecessary edgelordery. imo the messaging belongs, at best, in something like crazy cataclysm.
> 
> 
> 
> 

The only reason I didn't remove it in the PR myself is because I lack the technical skill to do so. FairyArmadillo mentioned it was hardcoded, so I assume I would need someone with C++ knowledge to help out?

**Comment by I-am-Erk** [comment]
I suggest calling it "uncaring".

The c++ should be just as easy as the JSON, it's just a matter of finding it

**Comment by fairyarmadillo** [comment]
> > personally, i'm in support of renaming _and_ removing the unnecessary edgelordery. imo the messaging belongs, at best, in something like crazy cataclysm.
> 
> The only reason I didn't remove it in the PR myself is because I lack the technical skill to do so. FairyArmadillo mentioned it was hardcoded, so I assume I would need someone with C++ knowledge to help out?

There are a several hooks for Psychopath all over the game, both in C++ and json. 

Consumption.cpp has a bunch of cannibalism messages that give you granular information about how you feel about eating people. These are edgy, but they're edgy because cannibalism is edgy, psychopath doesn't specifically make them so.

Construction.cpp adds some messaging about how it's fun to dig up graves if you're a psychopath.

Psychopaths get special messages for butchering people in activity_handlers, but none of it is edgy, it's all just like "you grit your teeth and get to work". They don't seem to enjoy it.

In JSON, there's some psychopath+cannibal dialog with Smokes (you can try to offer him human meat and act like it's the most normal thing in the world)

You can make a veiled threat to one of the refugee random encounter NPCs if you're a psychopath. It's kind of weird but not mega edgy.

There's a lot of text for psychopaths, but I'm not seeing much that's over the line or even anywhere near it. The smokes thing, I guess, and maybe the gravedigging thing could be converted to just not caring. Am I missing some, or are people conflating it with cannibal or killer drive?

**Comment by DukePaulAtreid3s** [comment]
> There's a lot of text for psychopaths, but I'm not seeing much that's over the line or even anywhere near it. The smokes thing, I guess, and maybe the gravedigging thing could be converted to just not caring. Am I missing some, or are people conflating it with cannibal or killer drive?

It's primarily the veiled threat that feels really video-gamey to me.

**Comment by fairyarmadillo** [comment]
> It's primarily the veiled threat that feels really video-gamey to me.

I'm not sure that there's any upside to removing the ability for the player to say weird things. If you were forced to pick the option or if it was some egregious act that stepped way over the line, sure, but it's just a moderately strange thing to say which prompts a realistic reaction from the NPC. People have said far weirder things to me IRL and we're not even in an apocalypse.

**Comment by fairyarmadillo** [comment]
I also don't see why the old description needs to go. There was nothing untoward in it, it was just an explanation of how the trait works.

**Comment by ashGlaw** [comment]
I dig the name change. Uncaring works for me

**Comment by KittyTac** [comment]
Moving to Killer Drive seems fine.

I think uncaring should be, yeah, just not giving a damn. Killer Drive should be the one that is actually violently unstable.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -1180,9 +1180,9 @@
   {
     "type": "mutation",
     "id": "PSYCHOPATH",
-    "name": { "str": "Psychopath" },
+    "name": { "str": "Uncaring" },
     "points": 2,
-    "description": "You don't experience guilt like others do.  Even when you know your actions are wrong, you just don't care.",
+    "description": "Whether due to brain chemistry, service-related trauma,or Olympian mental gymnastics,you no longer feel any remorse or guilt for harming your fellow man.",
```
Comment (Fris0uman) [inline data/json/mutations/mutations.json:1185]:
```suggestion
    "description": "Whether due to brain chemistry, service-related trauma, or Olympian mental gymnastics, you no longer feel any remorse or guilt for harming your fellow man.",
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -1180,9 +1180,9 @@
   {
     "type": "mutation",
     "id": "PSYCHOPATH",
-    "name": { "str": "Psychopath" },
+    "name": { "str": "Uncaring" },
     "points": 2,
-    "description": "You don't experience guilt like others do.  Even when you know your actions are wrong, you just don't care.",
+    "description": "Whether due to brain chemistry, service-related trauma, or Olympian mental gymnastics, you no longer feel any remorse or guilt for harming your fellow man.",
```
Comment (kevingranade) [inline data/json/mutations/mutations.json:1185]: I understand the desire to try to ground the effect somehow, but it's really not representative of anything. So we shouldn't be suggesting it is.
```suggestion
    "description": "You don't feel any remorse or guilt for harming your fellow man.",
```

**Comment by Consoleable** [comment]
I think Callous might be a more apt name. "Uncaring" is somewhat vague for my taste, but its not my PR.

**Comment by Relvl** [comment]
I read it, I laughed, thank you! Not sarcasm, it's fun to discuss psychopathy every time.
As a psychopath, I can explain some points.
Digging graves is not fun. Not terrible, but... What for? This description is more suitable for a sociopath, or some other deviations. As already mentioned above, we simply do not have the opportunity to feel empathy for other people, as well as some of the inhibitory mechanisms of the brain are weakened. Most psychopaths are well aware of their differences from other people, and know how to use it. All our lives we see the differences between ourselves and others. We learn to copy these emotions and reactions so as not to look like white crows. This helps us better understand the nature of lies, manipulation, and other ways of using people and ourselves.
Perhaps the best joke about psychopaths is "if a person is in trouble somewhere, I'll chalk his figure." It sounds strange in English, but it describes feelings well enough. It just doesn't matter if someone feels bad or good, as long as it doesn't directly affect me.

In other words, the name was exactly the right one. It has a description that helped to understand the meaning of this term to people who misunderstood it. Yes, some of the things in this perk sound strange to me, but overall it reflects reality.
